atlasdb-impl-shared/src/test/java/com/palantir/atlasdb/keyvalue/api/watch/LockWatchEventCacheIntegrationTest.java

```diff
@@ -287,11 +304,13 @@ public void nonContiguousEventsThrows() {
 
     @Test
     public void upToDateClientDoesNotThrow() {
+        eventCache = createEventCache(6);
         setupInitialState();
         eventCache.processStartTransactionsUpdate(TIMESTAMPS_2, SUCCESS);
-        assertThat(eventCache.getUpdateForTransactions(
-                TIMESTAMPS,
-                Optional.of(LockWatchVersion.of(LEADER, SUCCESS_VERSION))).events()).isEmpty();
+        assertThatCode(() -> eventCache.getUpdateForTransactions(
+                ImmutableSet.of(START_TS, 16L),
```

Comment:
I'm assuming you just wanted to make this test do a bit more by adding the extra timestamp to lookup?

Reply:
Yes